diff -Naru ghostscript-9.52_orig/freetype/src/base/ftobjs.c ghostscript-9.52/freetype/src/base/ftobjs.c
--- ghostscript-9.52_orig/freetype/src/base/ftobjs.c	2020-03-19 17:21:42.000000000 +0900
+++ ghostscript-9.52/freetype/src/base/ftobjs.c	2020-03-20 12:27:06.329844135 +0900
@@ -4060,7 +4060,7 @@
 
   FT_EXPORT_DEF( FT_UInt )
   FT_Get_Name_Index( FT_Face           face,
-                     const FT_String*  glyph_name )
+                     FT_String*  glyph_name )
   {
     FT_UInt  result = 0;
 
